.drone.yml

```diff
@@ -0,0 +1,18 @@
+workspace:
+  base: /go
+  path: src/github.com/gomods/athens
+
+pipeline:
+  ping:
+    image: mongo:3.0
+    commands:
+      - sleep 15
+  build:
+    image: golang:1.10
+    commands:
+      - go test ./...
+
+services:
+  mongo:
+    image: mongo:3.0
+    command: [ --smallfiles ]
```

Comment:
can you add a badge or link to where the drone page is? even better would be to add PR statuses from drone if possible

Reply:
Yes but it has to be enabled before I can add them so separate PR.